### PR TITLE
Minor revisions to the README.md and DevReadMe.md

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: DevReadMe.md [hunk 1]
```diff
@@ -100,17 +108,19 @@ ACA-Py ships with both inbound and outbound transport drivers for `http` and `ws
 
 Most configuration parameters are provided to the the agent at startup. Refer to the `Running` sections above for details on listing the available command line parameters.
```
Comment (loneil) [inline DevReadMe.md:109]: Not in the changes but an extra `the` here 😄

Comment (swcurran) [reply DevReadMe.md]: Thanks - fixed.

File: README.md [hunk 2]
```diff
@@ -7,11 +7,14 @@
 
 > An easy to use Aries agent for building SSI services using any language that supports sending/receiving HTTP requests.
 
+Full access to an organized set of all of the ACA_Py documents is available at [https://aca-py.org](https://aca-py.org).
```
Comment (usingtechnology) [inline README.md:10]: Have an underscore in ACA_Py

**Review by usingtechnology — APPROVED** [review]
Looks good, one minor typo in "ACA_Py" as noted. I like secure storage vs wallet.

File: README.md (hunk 2 above)
Comment (swcurran) [reply README.md]: Good catch — fixed.

**Pull request merged**